components/molecules/TableRepositoryName/table-repository-name.tsx

```diff
@@ -27,9 +26,12 @@ const TableRepositoryName = ({ avatarURL, fullName }: TableRepositoryNameProps):
 
       {/* Text */}
       <div className="flex flex-col justify-center">
-        <div title={name} className="  text-base text-light-slate-12 tracking-tight">
+        <div
+          title={name}
+          className="  text-base text-light-slate-12 tracking-tight max-w-[85px] sm:max-w-[150px] md:max-w-[110px] whitespace-nowrap overflow-hidden overflow-ellipsis"
```

Comment:
if you must use arbitrary value for sizes... use the `rem` equivalent 
```suggestion
          className="  text-base text-light-slate-12 tracking-tight max-w-[5.5rem] sm:max-w-[9.3rem] md:max-w-[7rem] whitespace-nowrap overflow-hidden overflow-ellipsis"
```

Reply:
@isabensusan, @OgDev-01, are these the standard widths for the app for this kind of field, or are these values arbitrary?

Reply:
hey! those are arbitrary, no idea where they come from

Reply:
as far as the table goes, i think this width works... we've been using arbitrary values to specify the widths